model/src/main/java/org/cbioportal/model/AlterationCountByGene.java

```diff
@@ -12,7 +12,7 @@
 	private Integer totalCount;
 	private Integer numberOfSamplesProfiled;
     private BigDecimal qValue;
-    private List<GenePanel> matchingGenePanels;
+    private List<String> matchingGenePanelIds;
```

Comment:
@zhx828 Is there a possibility for duplicate ids. If yes then better to use `Set`.

Reply:
The way the matchingGenePanelIds generated would avoid the duplication. But I agree the Set probably better here.

Reply:
@kalletlak updated.